test/integration/puppetlabs/services/jruby/jruby_pool_int_test.clj

```diff
@@ -37,6 +37,80 @@
    :ssl-key     localhost-key
    :ssl-ca-cert ca-cert})
 
+(def script-to-check-if-constant-is-defined
+  "begin; InstanceID; true; rescue NameError; false; end")
```

Comment:
I'll play with some ideas around the WP violation after we get the more important details squared away.